src/trio/_threads.py

```diff
@@ -475,8 +475,8 @@ def abort(raise_cancel: RaiseCancelT) -> trio.lowlevel.Abort:
                 msg_from_thread.run_sync()
             else:  # pragma: no cover, internal debugging guard TODO: use assert_never
                 raise TypeError(
-                    "trio.to_thread.run_sync received unrecognized thread message {!r}."
-                    "".format(msg_from_thread)
+                    f"trio.to_thread.run_sync received unrecognized thread message {msg_from_thread!r}."
+                    ""
```

Comment:
Extraneous empty string (and might collapse on one line without it).

Reply:
Removed in ec4c012